ci-operator/step-registry/cucushift/hypershift-extended/install-private/config/cucushift-hypershift-extended-install-private-config-ref.yaml

```diff
@@ -6,8 +6,10 @@ ref:
     tag: upi-installer
   env:
   - name: HYPERSHIFT_AWS_REGION
```

Comment:
you replaced  HYPERSHIFT_AWS_REGION with REGION, but still keep the definition here?